lib/vagrant-ovirt3/action.rb

```diff
@@ -13,7 +13,9 @@ def self.action_up
           b.use ConnectOVirt
           b.use Call, IsCreated do |env, b2|
             if env[:result]
-              b2.use MessageAlreadyCreated
+              b2.use StartVM
```

Comment:
This change is to handle the case where the VM has been created, but is halted.  If the machine is already up an error message from rbovirt is displayed.  I'm realizing that the exit code is probably different in that case, compared to the same operation without this change.

Would you like me to add a check to distinguish between a created but halted VM, versus a created and running VM?

Would you like me to restructure this to avoid duplication with lines 30-32?